sudoku/general/onInputCheck.js

```diff
@@ -0,0 +1,25 @@
+let wrapper = document.querySelector('.board');
+let check = document.querySelector('.check');
+wrapper.oninput = (event) => {
+  const regex = /^\d$/;
+
+  if (!event.target.innerHTML.match(regex)) {
+    event.target.style.backgroundColor = 'red';
+    check.setAttribute('disabled', 'disabled');
+  } else {
+    event.target.style.backgroundColor = 'white';
+    check.removeAttribute('disabled', 'disabled');
+  }
+};
+let hard = document.querySelector('.hard');
+hard.oninput = (event) => {
+  const regex = /^\d$/;
```

Comment:
```suggestion
  const REGEX_ONLY_DIGIT = /^\d$/;
```

It would be better to have all the regexps you are using in one place, so you can reuse them. You can create a config file and put all of them there, then import whenever you need them